debugger/src/debug_adapter/dap_adapter.rs

```diff
@@ -193,7 +194,7 @@ impl<P: ProtocolAdapter> DebugAdapter<P> {
                 ))),
             );
         };
-        let data_bytes = match base64::decode(&arguments.data) {
+        let data_bytes = match general_purpose::STANDARD.decode(&arguments.data) {
```

Comment:
Is there another way to write this? This seems very unclear, because base64 is not mentioned here anymore.

Reply:
True, it is lost in the `use` statement for `general_purpose`. Does this work for you?